models/EmailWhitelist.php

```diff
@@ -100,13 +100,18 @@ public static function getWhitelists()
         public static function emailIsAllowed($email)
         {
                 $allowed = self::getWhitelists();
-		$domain = explode('@', $email);
+		        $domain = explode('@', $email);
                 $domain = array_pop($domain);
                 $domain = strtolower($domain);
-                if (in_array($domain, $allowed)) { // email not whitelisted
+                if (in_array($domain, $allowed)) {
```

Comment:
@katka-n If that `if-else-if` looks strange. I guess it can be simplified and rewritten to:
```
if (in_array($domain, $allowed)) {
    return true;
}
if (in_array(strtolower($email), $allowed)) {
    return true;
}
return false;
```

Going on:
```
if (in_array($domain, $allowed) or in_array(strtolower($email), $allowed))
    return true;
return false;
```

But I'm not sure. I don't know PHP. However if that can be simplified in,any of 2 ways above and is a 1 minute thing, let's do it.